html/search-results.html

```diff
@@ -10,14 +10,14 @@
                     <h1>Yoda</h1>
                 </center>
             </div>
-            <div>
+            <div style="position: relative; left:500px; top:100px;">
               <p> 
                     Wize green dude who trains Luke Skywalker.
-                    Height:
-                    Mass:
-                    Gender:
-                    Skin Color:
-                    Homeworld:
+                    <p>Height:</p>
+                    <p> Mass:</p>
+                    <p>Gender:</p>
+                    <p> Skin Color:</p>
+                    <p>Homeworld:</p>
```

Comment:
I don't think these need to be in separate paragraph tags for each one since they are already within a parent paragraph tag. Or maybe you meant for there to be no parent tag at all?